index.js

```diff
@@ -218,10 +221,24 @@ function updateSchemesAndHost(req) {
   }
 }
 
+/**
+ * @description Generates definitions spec from mongoose models
+ *
+ * @returns Definitions spec
+ */
+function updateDefinitionsSpec(mongooseModels) {
+  const validMongooseModels = Array.isArray(mongooseModels) && mongooseModels.length > 0;
+  
+  if (validMongooseModels && !mongooseModelsSpecs) {
+    mongooseModelsSpecs = generateMongooseModelsSpec(mongooseModels);
+  }
+  return mongooseModelsSpecs;
```

Comment:
Is there any place where value returned from that functino is used? Maybe that return is not needed?

Reply:
Nope, it wasn't used. Returning void right now.